guides/SERVICE_NOW_SETUP.md

```diff
@@ -0,0 +1,44 @@
+# ServiceNow PDI (Personal Development Instance) Setup Guide
+
+- This guide describes the steps required in order to create a PDI (Personal Development Instance) that can be used in order to test the Blueprint's integration with ServiceNow.
+- Creating a PDI is an offered for free (at least for the time of writing this doc).
+
+## Step 1 - Signup + PDI New Instance
+1. First you should [sign up](https://signon.servicenow.com/x_snc_sso_auth.do?pageId=sign-up) to ServiceNow's Developer hub.
+
+2. Login into the developer hub -> Click "Request Instance" -> Select "Yokohama" as the instance release version. Note: it may take a few minutes until your new instance is available.
+
+3. Click on "Instance URL" in order to log in to your new instance as a admin (you can always go back to this link in case you forget your instance credentials).
+
+## Step 2 - Create a new "PC Refresh" Service Catalog
+1. Click "All" in the top left corner -> Search for "Catalog Builder" -> Click the link (should open the build in a new tab).
+2. Click "Build from scratch" -> Select "Standard" template -> Click "Continue"
+3. Configure the following under each section:
+### Details
+- Item name: PC Refresh
+- Short description: Flow to request a PC refresh
+### Location
+- Catalogs -> Browse -> Select "Service Catalog" -> Click "Save selection"
+- Categories -> Browse -> Select "Laptops", "Hardware", "Hardware Asset" -> Click "Save selection"
```

Comment:
I don't seem to have an option beyond "Laptops" 
<img width="1475" height="972" alt="image" src="https://github.com/user-attachments/assets/2a8846b8-0484-4a39-bb3c-8c72efeb5016" />